aloha-core/src/main/scala/com/eharmony/aloha/audit/Auditor.scala

```diff
@@ -0,0 +1,21 @@
+package com.eharmony.aloha.audit
+
+import com.eharmony.aloha.id.ModelIdentity
+
+/**
+  * Created by ryan on 1/11/17.
+  */
+trait Auditor[U, -N, +B <: U] {
```

Comment:
Would it be possible to lengthen how the type parameters are named?  Scala does emphasize brevity, but I've always found it a little difficult to decipher what the type parameters are from first glance.